state/s3.yaml

```diff
@@ -189,7 +240,28 @@ Resources:
           Principal:
             Service: 'config.amazonaws.com'
           Action: 's3:PutObject'
-          Resource: !Sub '${Bucket.Arn}/*AWSLogs/${AWS::AccountId}/Config/*'
+          Resource: !Sub '${Bucket.Arn}/*AWSLogs/${AWS::AccountId}/*'
+          Condition:
+            StringEquals:
+              's3:x-amz-acl': 'bucket-owner-full-control'
+  BucketPolicyConfigCrossAccountWrite:
```

Comment:
This allows any AWS account to publish their logs to your AWS account, correct? Is there no way to restrict based on OU/Account Id/...?